services/hanke-service/Dockerfile-platta

```diff
@@ -0,0 +1,24 @@
+# Uses different base images for build and run. RedHat Ubi stronger security policies prevent gradle builds.
+
+# Stage 1: Build application
+FROM eclipse-temurin:17 as build
+
+# Set the working directory, copy gradle and source code
+WORKDIR /app
+COPY gradlew settings.gradle.kts ./
+COPY gradle/ ./gradle/
+COPY services/hanke-service/build.gradle.kts ./services/hanke-service/
+COPY services/hanke-service/src ./services/hanke-service/src
+COPY scripts/wait-for-it.sh .
```

Comment:
I don't think we need this script for anything? It's used in Docker compose to wait for e.g. database to start before starting the backend service, but the database and other services are running constantly on Platta.

Reply:
I had a hunch that it may not be needed, but it was included in the original Dockerfile. Removed that line.

Reply:
There's another row with this, which would probably fail now that the other was removed.
```
COPY --from=build /app/wait-for-it.sh .
```

Reply:
Thanks, missed that one.